validator/project_validator.go

```diff
@@ -1898,3 +1899,26 @@ func parseS3PullParameters(c model.PluginCommandConf) (task, bv string, err erro
 	}
 	return task, bv, nil
 }
+
+// checkTaskTimeout checks if all tasks contain a timeout
+func checkTaskTimeout(project *model.Project) ValidationErrors {
+	errs := ValidationErrors{}
+	if project.ExecTimeoutSecs > 0 {
+		return errs
+	}
+	for _, task := range project.Tasks {
+		if task.ExecTimeoutSecs == 0 {
+			errs = append(errs,
+				ValidationError{
+					Message: fmt.Sprintf("task '%s' in project '%s' does not "+
+						"have an exec_timeout_secs defined; the task will default to a timeout of 6 hours\n",
```

Comment:
We should make this message more generic, i.e. something like "project %s has tasks without exec_timeout_secs defined; these tasks will default to a timeout of6 hours" since we are just checking if the project has ANY tasks without this specified to avoid spitting out a bunch of errors.

Reply:
Also I don't think we need the newline character - lets remove it. I dont see it on any other validation error messages.